src/main/java/org/llorllale/cactoos/matchers/EndsWith.java

```diff
@@ -53,7 +53,9 @@ public EndsWith(final Text text) {
             new TextMatcher(
                 new MatcherOf<>(
                     (String act) -> act.endsWith(text.asString()),
-                    text
+                        desc -> desc.appendText("ends with the word " + new Quoted(text)),
```

Comment:
@ruca905 please do not use the `+` but the already available methods on desc: `desc.appendText("ends with the word ").appendValue(text.asString())` which is responsible of quoting things properly.